python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -443,7 +422,8 @@ def shutdown(self):
         self.locked = set()
         try:
             global session_lock_refresher
-            session_lock_refresher.stop()
+            if session_lock_refresher and not session_lock_refresher.should_stop():
+                session_lock_refresher.stop()
```

Comment:
Is there any harm in calling `session_lock_refresher.stop()` if it's already been called?